app/sentry.ts

```diff
@@ -1,17 +1,24 @@
 import * as Sentry from '@sentry/browser';
-import { Ember } from '@sentry/integrations';
+import { CaptureConsole, Dedupe, Ember } from '@sentry/integrations';
 import config from 'open-event-frontend/config/environment';
 
 Sentry.init({
-    integrations: [new Ember()],
+    integrations: [
+        new Ember(),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 8.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5114150)